server/containers/images/Makefile

```diff
@@ -0,0 +1,149 @@
+# Base Makefile for building all images and tagging them
+
+# By default we only build images for x86_64 architectures.
+_ARCH = x86_64
+
+# By default we use this user's account on Fedora COPR for where to
+# find the RPMs.  You can override this using an environment variable
+# as appropriate.
+USER = ndokos
+
+# By default we use Fedora COPR repos. You can override this default
+# using an environment variable as appropriate.
+URL_PREFIX = https://copr-be.cloud.fedoraproject.org/results/${USER}
+
+# By default we use non-"test" COPR repos named "pbench".  We expect
+# test COPR repos to have a suffix added, typically "test", so that
+# the final repo name would be "pbench-test".
+TEST =
+
+# By default we use the pbench Quay.io organization for the image
+# repository.  You can override this default using an environment
+# variable as appropriate.
+IMAGE_REPO = docker://quay.io/pbench
+
+# Convenience reference to the repo template in the pbench tree.
+# Not intended to be overridden with an environment variable.
+_REPO_TEMPLATE = ../../ansible/roles/pbench-repo-install/templates/etc/yum.repos.d/pbench.repo.j2
+
+# The list of RPMs which provide the various tools we offer.
+# Not intended to be overridden with an environment variable.
+_TOOL_RPMS = pbench-sysstat perf blktrace bpftrace numactl pcp-system-tools strace kernel-tools tcpdump
+
+# The list of RPMs for the default workloads we offer.
+# Not intended to be overridden with an environment variable.
+_WORKLOAD_RPMS = fio uperf
+
+# Not intended to be overridden with an environment variable.
+_ALL_RPMS = ${_TOOL_RPMS} ${_WORKLOAD_RPMS}
+
+_DISTROS = centos-8 centos-7 fedora-32 fedora-31
+
+# Default we only build images the following distributions:
+all: $(foreach distro, ${_DISTROS}, ${distro}-base-tagged)
+
+#+
+# Tagging targets
+#-
+
+# Add the "latest" tag to the local images.
+tag-latest: $(foreach distro, ${_DISTROS}, ${_DISTROS}-tag-latest)
+
+# Add the "v<Major>-latest" tag to the local images.
+tag-major: $(foreach distro, ${_DISTROS}, ${_DISTROS}-tag-major)
+
+# Add the "v<Major>.<Minor>-latest" tag to the local images.
+tag-major-minor: $(foreach distro, ${_DISTROS}, ${_DISTROS}-tag-major-minor)
+
+#+
+# Push targets
+#-
+
+# Push images with "<git commit hash>" and "v<full RPM version>" tags.
+push: $(foreach distro, ${_DISTROS}, ${_DISTROS}-push)
+
+# Push images with the "latest" tag.
+push-latest: $(foreach distro, ${_DISTROS}, ${_DISTROS}-push-latest)
+
+# Push images with the "v<Major>-latest" tag.
+push-major: $(foreach distro, ${_DISTROS}, ${_DISTROS}-push-major)
+
+# Push images with the "v<Major>.<Minor>-latest" tag.
+push-major-minor: $(foreach distro, ${_DISTROS}, ${_DISTROS}-push-major-minor)
+
+#+
+# For the following rule patterns, the "%" represents the "distribution" name,
+# as derived from the "all" target's *-distro list.
+#-
```

Comment:
Maybe add a comment explaining `$*` as well.

Reply:
Wait, you asked me to change to use `$*`, and then you want me to explain that voodoo?!  How 'bout a little help with the wording in that existing comment ... ?

Reply:
You use the magic, you own the magic!

Reply:
Ah "Sourcerers Apprentice" here [pun intended] ...

Reply:
Applied some kind of comment, hopefully it makes sense.